modules/acquia_lift_publisher/src/EventSubscriber/Cdf/EntityRenderHandler.php

```diff
@@ -183,12 +183,10 @@ public function onCreateCdf(CreateCdfEntityEvent $event) {
           if (isset($view_modes['acquia_lift_preview_image'])) {
             $preview_image = $entity->{$view_modes['acquia_lift_preview_image']}->first();
 
-            if (!$preview_image) {
-              continue;
+            if ($preview_image) {
+              $src = ImageStyle::load('acquia_lift_publisher_preview_image')->buildUrl($preview_image->entity->getFileUri());
```

Comment:
Looks good. I wonder if we should also add another level of conditional to make sure that the imagestyle can actually build a valid URL as opposed to an empty string. Not sure how confident we can be that given an entity, buildUrl() will work.

Reply:
in other words, not add the preview_image attribute to cdf if we have an empty `$src`.

Reply:
Noted. I came up with something a bit different to hopefully protect further.